client/jetpack-cloud/sections/pricing/sale-banner/index.tsx

```diff
@@ -62,9 +62,12 @@ const SaleBanner: React.FC< Props > = ( { coupon } ) => {
 						<div>
 							<b>{ saleTitle }</b>
 							&nbsp;
-							{ translate( 'Take %(discount)d%% off all new annual Jetpack purchases.', {
-								args: { discount: coupon.final_discount },
-							} ) }
+							{ translate(
+								'Take %(discount)d%% off new annual Jetpack Security and Complete purchases.',
+								{
+									args: { discount: coupon.final_discount },
```

Comment:
I know it wasn't there before, but this could use a comment like you did [here](https://github.com/Automattic/wp-calypso/pull/76842/files)

Reply:
Makes sense, adding it 👍